projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3539,17 +3540,29 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
 
         this.resizeNotify.pipe(
             filter(() => !this._init),
-            throttleTime(40, animationFrameScheduler, { leading: false, trailing: true }),
+            throttleTime(40, animationFrameScheduler, { leading: true, trailing: true }),
```

Comment:
Is the `leading:true` necessary?  I think we removed it intentionally as part of a fix for the resize observer issue: https://github.com/IgniteUI/igniteui-angular/issues/13000
In PR: https://github.com/IgniteUI/igniteui-angular/pull/13684